tools/coqdoc/main.ml

```diff
@@ -379,23 +379,23 @@ let index_module = function
     Index.add_module m
   | Latex_file _ -> ()
 
+module E = Boot.Env
+
 let copy_style_file file =
   (* We give preference to coqlib in case it is overriden *)
-  let src_dir = CPath.choose_existing
-      [ CPath.make [ !Cdglobals.coqlib_path; "tools"; "coqdoc" ]
-      ; CPath.make [ !Cdglobals.coqlib_path; ".."; "coq-core"; "tools"; "coqdoc" ]
-      ] |> function
-    | None ->
-      eprintf
-        "coqdoc: cannot find coqdoc style files in coqlib: %s / %s\n"
-        !Cdglobals.coqlib_path file;
+  let env = E.init () in
+  let coqdoc = E.tool env "coqdoc" in
+  let sty_file = E.Path.relative coqdoc file in
+  if not (E.Path.exists sty_file) then
+    begin
+      let sty_file = E.Path.to_string sty_file in
+      eprintf "coqdoc: cannot find coqdoc style file: %s\n" sty_file;
       exit 1
-    | Some f -> f
-  in
-  let src = (CPath.relative src_dir file :> string) in
+    end;
   let dst = coqdoc_out file in
-  if Sys.file_exists src then FileUtil.copy src dst
-  else eprintf "Warning: file %s does not exist\n" src
+  let sty_file_s = E.Path.to_string sty_file in
+  if E.Path.exists sty_file then FileUtil.copy sty_file_s dst
```

Comment:
This if and the if above seem redundant.

Reply:
Indeed, thanks.